tests/test_pay.py

```diff
@@ -5395,3 +5395,31 @@ def test_delpay_works(node_factory, bitcoind):
                       status=failed['status'],
                       groupid=failed['groupid'],
                       partid=failed['partid'])
+
+
+def test_fetchinvoice_with_no_quantity(node_factory):
+    """
+    Reproducer for https://github.com/ElementsProject/lightning/issues/6089
+
+    The issue is when the offer has the quantity_max and the parameter.
+
+    In particular, in the fetchinvoice we forget to map the
+    quantity parameter with the invoice request quantity field.
+    """
+    l1, l2 = node_factory.line_graph(2, wait_for_announce=True,
+                                     opts={'experimental-offers': None,
+                                           'may_reconnect': True,
+                                           'dev-no-reconnect': None})
```

Comment:
You don't need these two I think, which is just as well, since dev-no-reconnect means the test needs to be marked DEVELOPER-only.

Reply:
ops! copy and paste error!